edge/pkg/edged/pleg/generic.go

```diff
@@ -118,7 +118,7 @@ func (gl *GenericLifecycle) convertStatusToAPIStatus(pod *v1.Pod, podStatus *kub
 
 	hostIP, err := gl.getHostIPByInterface()
 	if err != nil {
-		log.LOGGER.Errorf("Unable to get host IP")
+		log.LOGGER.Errorf("Cannot get host IP: %v", err)
```

Comment:
Thanks for add err msg! How about `Failed to get host IP`?

Reply:
Done @fisherxu